### DISCUSSION TITLE
Parental Controls: Daily Limit

### DISCUSSION THREAD

**Discussion by drKnoxy** [OnionUI/Onion#1232] [Ideas]
I have a sim card set up for my 8 year old, and I'd love to impose a daily limit parental control.

My thoughts on a feature:
1. config file for setting a daily limit; 15m m,t,w,th,fr 1hr sat,sun
2. when time is up, save state and pop a lock screen that says. "Saved! What was your favorite part of the last hour?"
4. (optional) "override password" that would extend play for 15 minutes

It seems like this might not be too far of a stretch from the Time Tracker app and enabling the WiFi clock that blocks the start.
I'd glad to help develop if folks think this is do-able

cc @nil0bject - since I saw your name all over the git annotate for that package

**Comment by Starbrush** [comment]
It would be really cool to be added! Being a very useful tool for parents

**Comment by Linkinterteid** [comment]
Вот из-за таких ограничений в детстве я и стал игроманом, нет уж, не нужно ограничивать ничего, пускай спиногрызы играют сколько влезет! )

**Comment by scelfo** [comment]
Any updates here? I'm also very interested in this capability and would be happy to help contribute.

**Comment by olafweller** [comment]
Would really love this feature!

**Comment by gael4910** [comment]
I would really need this feature for my kids too !

**Comment by jscti** [comment]
+1 for this feature.

**Comment by naymore** [comment]
+1 for parental control.

Could be as simple as possible.
(That said, limiting sd card content would be nice too)

**Comment by RoiSoleil** [comment]
+1 too
